src/main/java/TaskManager.java

```diff
@@ -102,10 +104,10 @@ public static void unmark(String[] args) {
         }
         System.out.println("Nice! I've marked these tasks as not done yet\n");
         for (int i = 1; i < args.length; i++) {
-            int reminderID = Integer.parseInt(args[i]);
-            int arrayID = getArrayID(reminderID);
+            int taskID = Integer.parseInt(args[i]);
+            int arrayID = getArrayID(taskID);
             if(arrayID < 0 || arrayID >= tasks.size()) {
-                System.out.println("There is no " + reminderID + " reminder.");
+                System.out.println("There is no " + taskID + " task.");
                 continue;
             }
             tasks.get(arrayID).clearDone();
```

Comment:
Should be one level of indentation lesser

Reply:
Already fixed using IntelliJ formatting, thanks for reviewing.